components/SimpleSlider/SimpleSlider.js

```diff
@@ -79,8 +80,9 @@ SimpleSlider.propTypes = {
     helperText: PropTypes.string,
     required: PropTypes.bool,
     decimalScale: PropTypes.number,
+    thousandSeparator: PropTypes.bool,
```

Comment:
This property is sent to CustomTextField component. In that component, this property is of type bool or string. Modify here also to be of type string or bool

Reply:
Add this property to d.ts also

Reply:
I have done the suggested modifications.